### DISCUSSION TITLE
How to claim zero basis for a lot of coins?

### DISCUSSION THREAD

**Discussion by davidrudnick** [eprbell/rp2#133] [Q&A] [answered]
Due to incomplete basis records for some of my previously acquired crypto, I have a tax lot of coins that I would like to claim zero basis for. I attempted to create an in transaction with type BUY and fiat in equal to zero and this generated an error. I also could not set the spot price to zero. Is there a way to accomplish what I am trying to do?

**Comment by eprbell** [comment, marked as answer]
Thanks for reporting this. I hadn't anticipated a situation where the basis could legitimately be zero, but this sounds like just such a case: we should fix this. Meanwhile here's a workaround. You can use a very small number for the spot price or for fiat in (e.g. 0.000000001 or similar). Remember to document why you did this in the Note field, so that you'll remember the details in case of an audit. Hope this helps.
